apps/cookbook/src/app/examples/dropdown-example/examples/dropdown-right-aligned.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-dropdown-example-dropdown-right-aligned',
+  template: `<kirby-dropdown
+  [size]="size"
+  placeholder="Right aligned dropdown"
+  [items]="['Item 1','Item 2','Item 3','Item 4','Item 5']"
+  cardAlign="right"
+></kirby-dropdown>`,
+  styles: [
+    `kirby-dropdown {
+margin-left: 200px;
```

Comment:
Will this work on small screens? 🤔

Reply:
Good point

Reply:
Removed the separate cookbook entry -solving this